src/overlays/gamestates/ovl_daytelop/z_daytelop.h

```diff
@@ -8,13 +8,20 @@ void Daytelop_Destroy(GameState* thisx);
 
 typedef struct {
     /* 0x000 */ GameState common;
```

Comment:
I think we should start using `state` for these like OoT.

Reply:
Yeah, I forgot to change that. Thanks